lib/build-manager.js

```diff
@@ -204,7 +204,7 @@ class BuildManager {
 
     return this.tagger.wrap({ tag, spec, version },
       async () => {
-        const vers = await this.Version.get(`${name}@${version}`);
+        const vers = await this.Version.get({ name, version });
         const pack = await thenify(vers, 'forBuild', read);
```

Comment:
This will be `await this.Version.forBuild({ pkg: vars, read })` with the new models
```suggestion
        const pack = await this.Version.forBuild({ pkg: vars, read })
```